cadcdata/cadcdata/tests/test_core.py

```diff
@@ -287,17 +285,17 @@ def test_help(self):
 
         # get -h
         usage = \
-'''usage: cadc-data get [-h] [-V]
+'''usage: cadc-data get [-h]
                      [--cert CERT | -n | --netrc-file NETRC_FILE | -u USER]
                      [--host HOST] [--resourceID RESOURCEID] [-d | -q | -v] -a
                      ARCHIVE [-o OUTPUT] [--cutout CUTOUT] [-de] [--wcs]
                      [--fhead]
-                     fileID [fileID ...]
+                     fileName [fileName ...]
```

Comment:
Will users need to know if the file is .fz or .gz or not compress?  or will using file names that exclude the compression extension work?   I'm jus thinking that the word 'filename'  (why camel case, filename is a word) is misleading if the user is required to know the compression that was used.